data_generation/generate.py

```diff
@@ -117,53 +120,58 @@ def create_networks(arguments):
 
     subgraphing_method = get_subgraphing_method(arguments.subgraphing_method)
     # A starting point is any bus that is connected to a generator to ensure that subgraphs contain at least one generator
+
     starting_points = full_net.gen.bus
     i = 0
     while i < arguments.num_subgraphs:
         print(f"generating network {i + 1}")
-        
-        subgraph_length = np.random.randint(arguments.min_size, min(arguments.max_size, len(full_net.bus)))
-        initial_bus = starting_points[np.random.randint(0, len(starting_points))]
-        
-        if arguments.n_1:
-            subgraph_busses = list(full_net.bus.index)
-            downed_bus = np.random.randint(0, len(subgraph_busses))
-            del subgraph_busses[downed_bus]
-    
+        if(arguments.subgraphing_method == 'num_change'):
+            subgraph_net = subgraphing_method(copy.deepcopy(full_net), None, None)
```

Comment:
if the 2nd and 3rd parameter are never used lets just remove them, if sometimes they are use, add a default=None in the function